golang/x/relay/types/msgs.go

```diff
@@ -51,3 +53,133 @@ func (msg MsgSetLink) GetSignBytes() []byte {
 
 // Route should return the name of the module
 func (msg MsgSetLink) Route() string { return RouterKey }
+
+/***** IngestHeaderChain *****/
+
+// MsgIngestHeaderChain defines a IngestHeaderChain message
+type MsgIngestHeaderChain struct {
+	Signer  string          `json:"signer"`
```

Comment:
Change the `Signer` type to `sdk.AccAddress` in each message

then implement
```
// GetSigners implements Msg
func (msg MsgWHATEVER) GetSigners() []sdk.AccAddress {
	return []sdk.AccAddress{msg.Signer}
}```

Reply:
that's a bit simpler

Reply:
will have to change the argument type in `func NewWHATEVER` as well

Reply:
Much better